exercises/minesweeper/.deprecated

```diff
@@ -0,0 +1,3 @@
+**NOTE: This exercise has been deprecated**
+Your track should implement "flower-field" instead.
```

Comment:
Should or may?

Reply:
Of the eleven other deprecated exercises, the language all reads "should" (in .deprecated) so maybe stick with that for consistency.